### ISSUE TITLE
Error message "unknown key: C- run"

### ISSUE THREAD

**Issue by salcode** [salcode/ironcode-tmux#31]
When a new tmux session starts I get the message

```
.tmux.conf:55: unknown key: C- run
```

**Comment by salcode** [comment]
The problem seems to be this line

```
bind -n C-\ run "(tmux display-message -p '#{pane_current_command}' | grep -iq vim && tmux send-keys 'C-\\') || tmux select-pane -l"
```

we're using from the this article, [Seamlessly Navigate Vim and tmux Splits](https://thoughtbot.com/blog/seamlessly-navigate-vim-and-tmux-splits)

Note: this issue surfaced when I setup a new machine, so there are a number of new variables.

**Comment by salcode** [comment]
Resolved in #33
